app/views/home/design_category.html.haml

```diff
@@ -0,0 +1,49 @@
+= stylesheet_link_tag 'home.css'
+
+:css
+  html, body {
+    height: 100%;
+  }
+  body {
+    padding-top: 87px;
+    padding-bottom: 50px;
+    overflow: hidden;
+  }
+  .container-fluid.scrollable {
+    height: 100%;
+    background: #f0f0f0;
+    color: #4d4d4d;
+    margin-top: 50px;
+  }
+
+  .text-container{
+      margin: 0 20%;
+      display: flex;
+      flex-direction: column;
+      height: 400px;
+      justify-content: center;
+  }
+
+  .text-body{
+      text-indent: 30px;
+  }
+
+= render "layouts/nav"
```

Comment:
I think it'd be a lot cleaner if we use:

```haml
= render "layouts/nav", locals: { simplified_nav: true }
```

And then instead of using `is_design_category` logic in the nav template, just `simplified_nav`.

This would also play nicely with @qxh5696's #497 since it does essentially the same thing